lib/commands/logging.js

```diff
@@ -0,0 +1,40 @@
+import _ from 'lodash';
+import logger from '../logger';
+
+let commands = {};
+
+const SUPPORTED_LOG_TYPES = {
+  'syslog': 'System Logs - Device logs for iOS applications on real devices and simulators',
+  'crashlog': 'Crash Logs - Crash reports for iOS applications on real devices and simulators',
+  'performance': 'Performance Logs - Debug Timelines on real devices and simulators'
+};
+
+commands.getLogTypes = function () {
```

Comment:
That sounds good. I was moving in that direction as I worked on other commands, too.